mautic/mautic
# Redirect not working for landing pages

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? |  Y
| New feature? | 
| Automated tests included? | Y
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
We were using redirection type drop down values from config file. May be due to wrong value in config or any other issue landing pages redirection url was not working.
Now we have added redirection type list hard coded.

#### Steps to test this PR:
1. Create a landing page (leave unpublished)
2. Select redirect type 301 or 302 and add Redirect URL
3. Visit the landing page URL in incognito mode/different browser
4. Should be redirected to the redirect URL

#### Other areas of Mautic that may be affected by the change:
1. 
2. 

#### List deprecations along with the new alternative:
1. 
2. 

#### List backwards compatibility breaks:
1. 
2.
